scipy/optimize/cython_optimize/__init__.py

```diff
@@ -0,0 +1,138 @@
+"""
+Cython Optimize Zeros API
+=========================
+The following four zeros functions can be accessed directly from Cython:
+
+- ``bisect``
+- ``ridder``
+- ``brenth``
+- ``brentq``
+
+Import the module into Cython as follows::
+
+    from scipy.optimize cimport cython_optimize
+
+The Cython Optimize Zeros API could be used to "vectorize" a root finder given
+a single callback function and an array of input conditions using a for-loop or
+``prange`` in Cython.
+
+
+Callback Signature
+------------------
+The zeros functions in this module use a callback that takes a double with the
+scalar independent variable as the 1st argument and a user defined ``struct``
+with any extra parameters as the 2nd argument. ::
+
+    double (*callback_type)(double, void*)
+
+Import the module into Cython as follows::
+
+    from scipy.optimize.cython_optimize cimport zeros
```

Comment:
I think `zeros` shouldn't be in the API, it should just be `cython_optimize.brentq` etc.

Reply:
✔️ done